packages/scandipwa/src/component/FieldFile/FieldFile.component.js

```diff
@@ -92,6 +94,7 @@ export class FieldFile extends PureComponent {
                   { ...attr }
                   // eslint-disable-next-line @scandipwa/scandipwa-guidelines/jsx-no-props-destruction
                   { ...events }
+                  onClick={ resetFieldValue }
```

Comment:
Do you override the `click` event here?
What if we pass `onClick` prop through the `events` config?

Reply:
You pass `onClick` inside the component.
But maybe you could put it to the `events` config inside the container?
And please make a check - if onClick already is present, call it inside the new event handler.

Reply:
Please see the `onChange` handling inside the container for the example.

Reply:
Done!